classes/class-p4-post-campaign.php

```diff
@@ -397,14 +397,42 @@ public static function css_vars( array $meta ): array {
 
 			$css_vars = array_merge( $css_vars, self::get_footer_theme( $meta ) );
 			$css_vars = array_merge( $css_vars, self::get_passive_button_color( $meta, $defaults ) );
-			$css_vars = array_merge( $css_vars, self::get_body_font( $meta, $defaults ) );
+			$css_vars = array_merge( $css_vars, self::get_body_font( $meta ) );
```

Comment:
Sure we don't need the defaults?

Reply:
No, in fact the above call does not need the defaults either.